tools/summonerd/src/web.rs

```diff
@@ -62,7 +62,7 @@ async fn serve_woff2(filename: &str) -> impl IntoResponse {
         .status(StatusCode::OK)
         .header("Content-Type", "font/woff2")
         .header("Cache-Control", "public, max-age=3600") // Cache for 1 hour
-        .body(axum::body::Full::from(data))
+        .body(axum::body::Body::from(data))
```

Comment:
`axum::body::Full` was removed, and one is to use `Body` instead (according to their changelog)